docs/react/bootstrap.md

```diff
@@ -43,22 +43,31 @@ It will create a folder called `my-app` inside the current folder with the follo
 
 ```console
 my-app
-├── README.md
+├── .gitignore
 ├── node_modules
```

Comment:
Let's delete this, so that it aligns with the note.

```suggestion
```

Reply:
> Let's delete this, so that it aligns with the note.

Could you tell me where should the alignment be with the note?

Reply:
The note says we omit it. I committed the suggestion. This is good to merge. Thank you!